src/main/java/com/github/fppt/jedismock/operations/OperationFactory.java

```diff
@@ -71,20 +73,22 @@
         TRANSACTIONAL_OPERATIONS.put("smembers", RO_smembers::new);
         TRANSACTIONAL_OPERATIONS.put("hsetnx", RO_hsetnx::new);
         TRANSACTIONAL_OPERATIONS.put("time", RO_time::new);
+        TRANSACTIONAL_OPERATIONS.put("blpop", RO_blpop::new);
```

Comment:
That's weird. This was already supported. Github merge issues?

Reply:
weird indeed, might be because I added this command as well, sorry, I'm not expert in git and sometimes I might mix up things.